features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/wso2/templates/item-design/template.jag

```diff
@@ -649,6 +663,52 @@
 </div>  
 </script>
 
+<script id="designer-sequence-template" type="text/x-handlebars-template">
+<div class="resource_body_padding add-margin-bottom-2x">
+	<div class="add-margin-bottom-4x">
+    	<strong>Description : </strong><a class="notes" data-path="{{resource_path}}" data-attr="description" title="<%=i18n.localize("Description")%>">{{ description }}</a>
+	</div>
+	<div class="add-margin-bottom-4x">
+    	<strong>Response Content Type : </strong><a href="#" data-path="{{resource_path}}" data-attr="content_type" class="content_type" data-type="typeahead" data-pk="1" data-title="Responce Content Type" title="ResponseContent Type">{{ content_type }}</a></h5>
+    </div>
+    <div class="add-margin-bottom-4x">
+		<div class="add-margin-bottom-2x">
+			<strong>Parameters : </strong>
+		</div>
+		{{#if parameters}}
+		<table class="table table-condensed table-bordered">
+			<tr>
+				<th width="15%"><%=i18n.localize("Parameter Name")%></th>
+				<th width="40%"><%=i18n.localize("Description")%></th>
+				<th width="15%"><%=i18n.localize("Parameter Type")%></th>
+				<th width="15%"><%=i18n.localize("Data Type")%></th>
+				<th width="15%"><%=i18n.localize("Required")%></th>
+			</tr>
+		{{#each parameters}}
+			<tr>
+			<td>{{ name }}</td>
+			<td><a class="param_desc" data-path="{{ ../resource_path}}.parameters[{{@index}}]" data-attr="description" title="<%=i18n.localize("Description")%>">{{ description }}</a></td>
+			<td><a class="param_paramType" data-type="select" data-path="{{ ../resource_path}}.parameters[{{@index}}]" data-attr="in" title="in">{{ in }}</a></td>
+			<td><a class="param_type" data-path="{{../resource_path}}.parameters[{{@index}}]" data-attr="type" title="type">{{ type }}</a></td>
+			<td><a class="param_required" data-type="select" data-path="{{../resource_path}}.parameters[{{@index}}]" data-attr="required" data-value="{{toString required }}" title="required"></a></td>
+			</tr>
+		{{/each}}
+		{{/if}}
+		</table>
+  	</div>
+  	<div class="add-margin-bottom-4x">
+  		<div class="add-margin-bottom-2x">
+			<strong>Sequences : </strong>
```

Comment:
Shouldn't we use something like "Conversion Policies"? "Sequence" is a technical name used in synapse which has less meaning in UI level.